### PR TITLE
Updates from the webapp branch

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
General fixes/updates I made to the webapp branch. @enahum not sure if the view channel changes will affect RN. If they don't work let me know and I can make it support the old behaviour as well

Note that `loadMe` changed to just use all the actions instead of the client directly. If that's an issue for RN we can chat about it as well

### CODE REVIEW

File: src/actions/channels.js [hunk 1]
```diff
@@ -568,6 +561,8 @@ export function viewChannel(channelId) {
             return null;
         }
 
+        setServerVersion(Client4.getServerVersion())(dispatch, getState);
```
Comment (enahum) [inline src/actions/channels.js]: why this? there is an event that is fired anytime the serverVersion changes

Comment (jwilander) [inline src/actions/channels.js]: Does it update the store anytime it changes in the client? I wasn't seeing that happen so I just had it get set on a regular call like view channel

Comment (enahum) [inline src/actions/channels.js]: if you listen for the event you can use that to update the store accordingly

**Comment by jwilander** [comment]
@enahum fixed

**Pull request merged**